apps/site/src/views/Landing/components/ApplyButton/ApplyButton.module.scss

```diff
@@ -0,0 +1,33 @@
+@use "bootstrap-utils" as utils;
+
+.applyButtonContainer {
+	position: relative;
+}
+
+.applyButton {
+	position: relative;
+	transform: skew(-30deg);
+	height: 80px;
+	width: 250px;
+	margin: 0px;
+	border: none;
+	background-color: utils.$yellow;
+	transition: all 0.1s ease;
+
+	&:hover {
+		border: black solid 6px;
+		background-color: utils.$yellow;
```

Comment:
Thought: it may be better to define a proper button variant to handle the other states. Currently, the `:active` state reverts to Bootstrap's primary variant.

Reply:
How would I you like me to do this?

Reply:
I added a `warning` variant to the Button

Reply:
I just switched to regular button.

Reply:
Using a `button` for a link is not appropriate. I'll handle this.

Reply:
Is it not good practice to put an `<a>` tag wrapped around a `<button>` ?

Reply:
I switched to a `div` instead. I didn't know that `a` tags shouldn't wrap interactive content per HTML W3C rules. So, thanks for the catch! Feel free to request or make any more changes, though, if the changes are not adequate!